vividus-extension-selenium/src/main/java/org/vividus/selenium/WebDriverManagerContext.java

```diff
@@ -37,6 +40,19 @@ public <T> T getParameter(WebDriverManagerParameter parameter)
         return initialValueSupplier == null ? testContext.get(key) : testContext.get(key, initialValueSupplier);
     }
 
+    @Override
+    public <T> T get(WebDriverManagerParameter parameter, Supplier<T> initialValueSupplier)
+    {
+        if (parameter.getInitialValueSupplier() == null)
+        {
+            return testContext.get(parameter.getContextKey(), initialValueSupplier);
+        }
+        else
+        {
+            throw new IllegalArgumentException("Initial value supplier for parameter is not null");
```

Comment:
it makes sense to refer the parameter name in the exception message

Reply:
done